pymc_experimental/tests/test_distributions.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import scipy.stats
+import scipy.stats.distributions as sp
+from pymc_experimental.distributions import GeneralizedGamma
+from pymc.tests.test_distributions import Rplus, Rplusbig, check_logp, check_logcdf
+
+
+class TestMatchesScipy:
+    def test_generalized_gamma(self):
+        check_logp(
+            GeneralizedGamma,
+            Rplus,
+            {"alpha": Rplusbig, "p": Rplusbig, "lambd": Rplusbig},
+            lambda value, alpha, p, lambd: sp.gengamma.logpdf(value, a=alpha / p, c=p, scale=lambd),
+        )
+        check_logcdf(
+            GeneralizedGamma,
+            Rplus,
+            {"alpha": Rplusbig, "p": Rplusbig, "lambd": Rplusbig},
+            lambda value, alpha, p, lambd: sp.gengamma.logcdf(value, a=alpha / p, c=p, scale=lambd),
```

Comment:
How does scipy compute the logcdf?

Reply:
I looked closer, the difference in their calculation of the cdf is just related to how they chose to parameterize the distribution. 

Their parameterization seems like it might be less ideal, where they allow one of their parameters to be both negative and positive, but not zero, as opposed to the parameterization here which has the equivalent parameter as constrained positive. So not much to report here, I think not mimicking scipy makes sense here, but I also didnt test it out